app/views/users/following.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>following</h1>
+<% current_user.accepted_sent_follow_requests.each do |list| %>
```

Comment:
There is a more concise way to write this each statement. Review the user accessor methods in your user model.